contracts/fuelchain/FuelMessagePortal.sol

```diff
@@ -125,12 +121,6 @@ contract FuelMessagePortal is
         _unpause();
     }
 
-    /// @notice Sets the waiting period for message root states
-    /// @param messageTimelock The waiting period for message root states (in milliseconds)
-    function setIncomingMessageTimelock(uint64 messageTimelock) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
No need for the timelock anymore?

Reply:
The old timelock concept is now handled by the "TIME_TO_FINALIZE" parameter (one of the parameters that eventually will dictate the fault proving window).